website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -127,60 +126,44 @@ const authOptions: AuthOptions = {
       session.user.role = token.role;
       session.user.isNew = token.isNew;
       session.user.name = token.name;
+      session.user.image = token.picture;
```

Comment:
You can receive image from user property in the callback here

Reply:
you mean as a parameter to the session callback?

the `user` object is only passed if we are using the database strategy, but we are using jwt.

Or did I  misunderstand what you say?

Reply:
> the `user` object is only passed if we are using the database strategy, but we are using jwt.

Hmm I don't see where the docs mention this, the TS signature doesn't mark it as nullable as well.

Reply:
![image](https://github.com/LAION-AI/Open-Assistant/assets/24505302/6f0e72c6-c958-4f68-8159-d4965a12189b)

if I console log it is undefined, would be a lot easier if we did not have to pass everything though the token.